pkg/blobinfocache/memory/memory_test.go

```diff
@@ -7,6 +7,9 @@ import (
 	"github.com/containers/image/v5/types"
 )
 
+var _ types.BlobInfoCache = &cache{}
+var _ types.BlobInfoCache2 = &cache{}
```

Comment:
This would be useful to have for `NoCache` as well.

Reply:
I keep forgetting about `NoCache`.  Will add it.